rmw_test_fixture_implementation/CMakeLists.txt

```diff
@@ -88,14 +88,6 @@ if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
   set_target_properties(_rmw_test_fixture_implementation PROPERTIES DEBUG_POSTFIX "_d")
 endif()
 
-# Set output directories to import module from the build directory
-# Use a no-op generator expression so multi-config generators don't append an
-# extra directory like Release/ or Debug/ and break the Python import.
-set_target_properties(_rmw_test_fixture_implementation PROPERTIES
-  LIBRARY_OUTPUT_DIRECTORY "$<1:${CMAKE_CURRENT_BINARY_DIR}/test_${PROJECT_NAME}>"
-  RUNTIME_OUTPUT_DIRECTORY "$<1:${CMAKE_CURRENT_BINARY_DIR}/test_${PROJECT_NAME}>"
-)
-
```

Comment:
This block is changing the directory where the compiled library is placed under the `build` tree. Rather than putting it in a specific place where the tests can find it, I used a generator expression to tell the test to look for it wherever CMake decided to put it (`$<TARGET_FILE_DIR:_rmw_test_fixture_implementation>`).

Since there were no tests before, this wasn't being used, and was just copy/pasted from another project that was using `python3_add_library`.